src/rcdom.rs

```diff
@@ -303,6 +312,13 @@ impl TreeSink for RcDom {
             panic!("not a script element!");
         }
     }
+
+    fn is_mathml_annotation_xml_integration_point(&self) -> bool {
+        match (**self.document).borrow().node {
```

Comment:
The thing you need to check is the given `Handle` that needs to be passed to the method.